packages/gatsby-plugin-manifest/src/pluginOptionsSchema.js

```diff
@@ -0,0 +1,236 @@
+export default function pluginOptionSchema({ Joi }) {
+  /* Descriptions copied from or based on documentation at https://developer.mozilla.org/en-US/docs/Web/Manifest
+   *
+   * Currently based on https://www.w3.org/TR/2020/WD-appmanifest-20201019/
+   */
+
+  const platform = Joi.string()
+    .optional()
+    .empty(``)
+    .description(`The platform on which the application can be found.`)
+
+  const FingerPrint = Joi.object().keys({
+    type: Joi.string()
+      .required()
+      .description(`syntax and semantics are platform-defined`),
+    value: Joi.string()
+      .required()
+      .description(`syntax and semantics are platform-defined`),
+  })
+
+  const ManifestImageResource = Joi.object().keys({
+    sizes: Joi.string()
+      .optional()
+      .description(`A string containing space-separated image dimensions`),
+    src: Joi.string()
+      .required()
+      .description(
+        `The path to the image file. If src is a relative URL, the base URL will be the URL of the manifest.`
+      ),
+    type: Joi.string().description(
```

Comment:
Nit pick: this is missing `.optional` which I know is the default but it's explicit everywhere else